config/config/wc-config.yaml

```diff
@@ -22,53 +27,95 @@ storageClasses:
   ebs:
     enabled: null # true | false
 
+  storageClass: "set-me"
+
+  ## IP of the cluster DNS in kubernetes
+  ##
+  clusterDns: "10.43.0.10"
+
+## Object storage configuration for backups.
+##
 objectStorage:
-  # Options are "s3", "gcs", or "none"
-  # If "none", remember to disable backups (velero)
+  ## Options are "s3", "gcs", or "none"
+  ## If "none", remember to disable backups (velero)
   type: "s3"
   # gcs:
   #   project: "set-me"
   # s3:
   #   region: "set-me"
   #   regionAddress: "set-me"
   #   regionEndpoint: "set-me"
+
+  ##
+  ## TODO: remove all but for 'velero'.
```

Comment:
Why remove those?

Reply:
Because only the velero bucket is used in the workload cluster.

Reply:
Ofcorse, I didn't think about that.

Reply:
Could you remove these now instead of just adding a TODO? If they are not used anywhere I don't think it should be a problem.

Reply:
Ok